source/agora/node/Validator.d

```diff
@@ -99,9 +99,6 @@ public class Validator : FullNode, API
     /// The identity we present on `handshake`
     private Identity identity;
 
-    /// Signatures we received a early
-    private Set!ValidatorBlockSig early_sigs;
```

Comment:
These helped with signatures arriving before SCP signals us the externalization event. I think that is still possible even if we delay the actual externalization, no?

Reply:
Yes that is true. I'll revert this change and store signatures these early signatures.

Reply:
Moved them to `Nominator` and store them until block is pending.